app/admin/top-issues/components/AdminTopIssuesPage.js

```diff
@@ -54,33 +59,37 @@ export default function AdminTopIssuesPage(props) {
         </BlackButtonClient>
 
         {addNewIssue && (
-          <div>
-            <br />
-            <br />
+          <div  className="flex mt-4 items-center" >
+            <SVGIconChooser {...{
+              svgData,
+              setSvgData
+            }} />
             <TextField
+              className="mx-4"
               fullWidth
               primary
               label="Top Issue Name"
               value={topIssueName}
               onChange={(e) => setTopIssueName(e.target.value)}
             />
-            <br />
-            <br />
             <div className="text-right">
               <BlackButtonClient
                 disabled={topIssueName === ''}
                 onClick={handleCreate}
                 className="font-black"
               >
```

Comment:
This is the old deprecated button.
Change to
```
        <div className="text-right"  onClick={handleCreate}>
              <PrimaryButton
                disabled={topIssueName === ''}               
              >
                 Save
              </PrimaryButton>
      </div>

```

Reply:
I didn't modify this code. You want me to do this refactor as part of this PR?  Seems like an update that might be more safely done in a separate ticket/PR, yes?

Reply:
My general approach is to fix it if we touch it.

Reply:
https://github.com/thegoodparty/gp-webapp/pull/122/commits/8d711b2745dc7d7fcfa7eab514c9588ec831cbd5